spec/services/campaign_page_builder_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe CampaignPageBuilder do
+  before do
+    allow(ChampaignQueue).to receive(:push)
```

Comment:
They don't. I might add a webmock expectation to test that the aws sdk gem makes the required http request when `push` is sent to `ChampaignQueue`. Presently I'm testing integration with `expect_any_instance_of(Aws::SQS::Client).to receive(:send_message).with(opts)`. I think this is enough.